packages/tasit-contracts/decentraland/scripts/migrate.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+PROJECT_DIR=$1
+if [ -z "$PROJECT_DIR" ];
+then
+    echo "Error! Use: ./$0 <tasit-contract-dir>";
+    exit;
+fi
+
+DECENTRALAND_DIR="$PROJECT_DIR/decentraland"
+
+# mana
+cp $DECENTRALAND_DIR/scripts/3_mana_token.js $DECENTRALAND_DIR/mana/migrations
+cd $DECENTRALAND_DIR/mana && npx truffle migrate
+
+# land
+cp $DECENTRALAND_DIR/scripts/3_land_registry.js $DECENTRALAND_DIR/land/migrations
+cd $DECENTRALAND_DIR/land && npx truffle migrate --network livenet
```

Comment:
What's the `--network livenet` for?

Reply:
`land` project has that network pointing to `8454` port of ganache, the default `develop` network is pointing to `18545` and isn't working.

Reply:
Ah ok. That might warrant a comment in the code